CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-rc.5 July 28, 2021
+
+**The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
+
+## Breaking changes ⚠️
+* Improve camera API consumption from java programming language by adding a JvmName annotations. ([#495](https://github.com/mapbox/mapbox-maps-android/pull/495))
```

Comment:
Does this PR only changes the Java signature? So it's a breaking change only for Java users?

Reply:
Yes, it should be relevant for Java only users.